### ISSUE TITLE
Fix Dialog action buttons are not aligned

### ISSUE THREAD

**Issue by Jekata** [telerik/kendo-ui-core#2442]
### Bug report

When the <link rel="stylesheet" href="//kendo.cdn.telerik.com/2016.3.1028/styles/kendo.mobile.all.min.css" /> file is inserted in a page with Kendo Dialog, the following CSS class 

```
 .km-nova [class^="k-"] {
     -webkit-box-sizing: content-box;
     box-sizing: content-box;
 }
```
overrides the dialog css and breaks the appearance of its action buttons:

[Dojo sample](http://dojo.telerik.com/@RumenJ/UpuYI/6)

### Reproduction of the problem
Go to and hit the run button: [Dojo sample](http://dojo.telerik.com/@RumenJ/UpuYI/6)

### Expected/desired behavior
The three buttons at bottom should be rendered on a single line.

### Environment

* **Kendo UI version:** 2016.3.1028
* **jQuery version:** all
* **Browser:** [all] 

forum thread: http://www.telerik.com/forums/dialog-action-buttons-in-mobile-app

**Comment by Jekata** [comment]
Workaround: 

```
<style>
 .km-nova [class^="k-"] {
   -webkit-box-sizing: border-box;
   box-sizing: border-box;
 }
  .k-window-titlebar.k-header { height: 49px; }
</style>
```

**Comment by xinthose** [comment]
This still is not fixed in v2017.1.118.
